astrology.lic

```diff
@@ -121,10 +124,29 @@ class Astrology
     waitrt?
   end
 
+  def roll_bones
+    if @divination_bones_tied
+      bput('untie bones from my '+@divination_bones_container, 'divination bones')
+    else
+      bput('get bones from my '+@divination_bones_container, 'divination bones')
+    end
+    bput('roll my bones', 'roundtime')
+    waitrt?
+    if @divination_bones_tied
+      bput('tie bones to my '+@divination_bones_container, 'divination bones')
+    else
+      bput('put bones in my '+@divination_bones_container, 'divination bones')
+    end
+  end
+
   def align(skill)
     bput("align #{skill}", 'You focus internally')
     waitrt?
-    bput('predict future', 'roundtime')
+    if @have_divination_bones
+      roll_bones
```

Comment:
Yes, it takes the place of `predict future`.  It actually is capable of performing a much stronger prediction than just `predict future` once you are bonded with the bones sufficiently (which makes the script great, because each roll increases the bond)